Tasks/AzureMysqlDeploymentV1/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 1,
         "Minor": 184,
```

Comment:
https://whatsprintis.it/
reports sprint as 185

Reply:
since it is getting deployed with M184 payload, is is better to use 184 only